app/imports/components/Users/AdminPanel/UsersManagement/UsersList/usersList.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Avatar from 'material-ui/Avatar';
+import Subheader from 'material-ui/Subheader';
+import { List, ListItem, makeSelectable } from 'material-ui/List';
+import { PropTypes } from 'prop-types';
+import SetAdmin from './setAdmin';
+
+
+export default class UsersList extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.SelectableList = makeSelectable(List); // @TODO Do it like in tutorial
+    }
+
+
+    render() {
+        return (
+            <div>
+                <this.SelectableList>
+                    <Subheader>Users</Subheader>
+                    {this.props.users.map(user => (
+                        <ListItem
+                            value={1}
```

Comment:
I asked about it, because I didn't find it in official documentation, but I went deeper...

...and I found, check and test it. You should also do it. You're using it in wrong way.

Case: user list - your selectable list acts like a standard `List` - I don't see any difference.
Case: project list - all items are selected.